# Study 894 — Trend Overlay on 60/40 — the teardown 📉

The excess-of-cash vs excess-of-cash race, the HAC *t* on the return difference, the **paired** Sharpe-advantage bootstrap, the two-era cut, the calendar table, the switching-cost grid, the short-term-gains tax drag, and the synthetic control.

In [1]:
R = {'start': '2007-05-30', 'end': '2026-06-30', 'n_days': 4602, 'fingerprint': '0dd2af7e1636', 'sharpe_strat': 0.823, 'sharpe_bench': 0.689, 'sharpe_adv': 0.134, 'maxdd_strat': -12.5, 'maxdd_bench': -30.8, 'dd_cut': 18.3, 'vol_strat': 6.9, 'vol_bench': 11.4, 'cagr_strat': 6.94, 'cagr_bench': 8.87, 'diff_bps': -0.87, 't_nw': -1.18, 't_1s': -0.96, 'boot_adv': 0.134, 'boot_lo': -0.242, 'boot_hi': 0.517, 'boot_ppos': 0.744, 'boot_diff': -0.87, 'boot_diff_lo': -2.23, 'boot_diff_hi': 0.59, 'era_early_adv': 0.204, 'era_early_dd': 21.3, 'era_early_n': 2217, 'era_late_adv': 0.071, 'era_late_dd': 8.5, 'era_late_n': 2385, 'y2008_ov': 2.1, 'y2008_st': -13.2, 'y2022_ov': -8.5, 'y2022_st': -16.4, 'cost3_adv': 0.104, 'cost5_adv': 0.084, 'tax15_adv': -0.09, 'tax25_adv': -0.285, 'tax25_cagr': 4.12, 'syn_plant_adv': 0.31, 'syn_plant_dd': 40.9, 'syn_plant_fire': 11, 'syn_null_adv': 0.04}

## The headline — overlay vs static 60/40 (excess of BIL, gross)

In [2]:
print(f"excess Sharpe : overlay {R['sharpe_strat']:.3f} vs static {R['sharpe_bench']:.3f}  "
      f"(advantage {R['sharpe_adv']:+.3f})")
print(f"max drawdown : {R['maxdd_strat']:+.1f}% vs {R['maxdd_bench']:+.1f}%  "
      f"(cut {R['dd_cut']:+.1f} pp)")
print(f"vol / CAGR   : {R['vol_strat']:.1f}% / {R['cagr_strat']:.2f}%  vs  "
      f"{R['vol_bench']:.1f}% / {R['cagr_bench']:.2f}%")
print(f"return diff  : {R['diff_bps']:+.2f} bps/day  NW(10) t = {R['t_nw']:+.2f}  "
      f"(one-sample t = {R['t_1s']:+.2f})")

excess Sharpe : overlay 0.823 vs static 0.689  (advantage +0.134)
max drawdown : -12.5% vs -30.8%  (cut +18.3 pp)
vol / CAGR   : 6.9% / 6.94%  vs  11.4% / 8.87%
return diff  : -0.87 bps/day  NW(10) t = -1.18  (one-sample t = -0.96)


## Is the Sharpe advantage real? Paired block-bootstrap

Resample overlay and static excess returns with the *same* block indices each draw and take the Sharpe difference — a CI for the advantage itself.

In [3]:
print(f"Sharpe advantage {R['boot_adv']:+.3f}  95% CI [{R['boot_lo']:+.3f}, {R['boot_hi']:+.3f}]  "
      f"P(adv>0) = {R['boot_ppos']:.3f}")
print(f"return diff     {R['boot_diff']:+.2f} bps/day  95% CI [{R['boot_diff_lo']:+.2f}, {R['boot_diff_hi']:+.2f}]")
print('-> both CIs straddle zero: the risk-adjusted OUTperformance is not robust.')

Sharpe advantage +0.134  95% CI [-0.242, +0.517]  P(adv>0) = 0.744
return diff     -0.87 bps/day  95% CI [-2.23, +0.59]
-> both CIs straddle zero: the risk-adjusted OUTperformance is not robust.


## Robustness — two eras (split 2017-01-01)

The advantage is front-loaded on 2008; post-2017 it fades.

In [4]:
print(f"2007-2016 (n={R['era_early_n']}): Sharpe adv {R['era_early_adv']:+.3f}, DD cut {R['era_early_dd']:+.1f} pp")
print(f"2017-2026 (n={R['era_late_n']}): Sharpe adv {R['era_late_adv']:+.3f}, DD cut {R['era_late_dd']:+.1f} pp")

2007-2016 (n=2217): Sharpe adv +0.204, DD cut +21.3 pp
2017-2026 (n=2385): Sharpe adv +0.071, DD cut +8.5 pp


## Calendar years — the edge is 2008 & 2022 (net of 3 bps switching)

In [5]:
print(f"2008: overlay {R['y2008_ov']:+.1f}%  static {R['y2008_st']:+.1f}%  (diff {R['y2008_ov']-R['y2008_st']:+.1f} pp)")
print(f"2022: overlay {R['y2022_ov']:+.1f}%  static {R['y2022_st']:+.1f}%  (diff {R['y2022_ov']-R['y2022_st']:+.1f} pp)")

2008: overlay +2.1%  static -13.2%  (diff +15.3 pp)
2022: overlay -8.5%  static -16.4%  (diff +7.9 pp)


## Costs are trivial, tax is the killer

A 200-day rule trades rarely, so switching costs barely move the edge — but each forced exit to cash realises a short-term gain a buy-and-hold book defers.

In [6]:
print(f"switching only  3bps: Sharpe adv {R['cost3_adv']:+.3f}   5bps: {R['cost5_adv']:+.3f}")
print(f"+ 15% ST tax        : Sharpe adv {R['tax15_adv']:+.3f}  (already negative)")
print(f"+ 25% ST tax        : Sharpe adv {R['tax25_adv']:+.3f}  (CAGR {R['tax25_cagr']:.2f}%)")

switching only  3bps: Sharpe adv +0.104   5bps: +0.084
+ 15% ST tax        : Sharpe adv -0.090  (already negative)
+ 25% ST tax        : Sharpe adv -0.285  (CAGR 4.12%)


## Synthetic positive control — the machinery is unbiased

Live: the overlay must recover a planted trend benefit and stay flat on the null.

In [7]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
import numpy as np
from trend6040 import data, strategy as st
adv_p = np.array([st.synthetic_detect(data.synthetic_prices(edge=1.0, seed=894+s, n_days=5000))['sharpe_adv'] for s in range(8)])
adv_n = np.array([st.synthetic_detect(data.synthetic_prices(edge=0.0, seed=894+s, n_days=5000))['sharpe_adv'] for s in range(8)])
print(f"planted (edge=1), 8 seeds: Sharpe adv mean {adv_p.mean():+.3f}, adv>0 in {(adv_p>0).sum()}/8")
print(f"null    (edge=0), 8 seeds: Sharpe adv mean {adv_n.mean():+.3f} (no bear to duck)")

planted (edge=1), 8 seeds: Sharpe adv mean +0.281, adv>0 in 7/8
null    (edge=0), 8 seeds: Sharpe adv mean -0.034 (no bear to duck)


## Verdict

- **Signal — Weak.** A **real, robust drawdown & vol cut** (max DD -12.5% vs -30.8%, positive in both eras) — but the *Sharpe advantage* (+0.13) has a bootstrap CI that straddles zero ([-0.24, +0.52], P>0 = 0.74), the return difference is negative & insignificant (-0.87 bps/day, NW *t* = -1.18), and it is front-loaded on 2008 (era adv +0.20 → +0.07). Real risk reduction, no robust risk-adjusted outperformance. *Short history flatters the trend rule.*
- **Tradability — Fragile.** Trading costs are trivial (+0.08 adv at 5 bps), so a tax-deferred account gets a genuinely calmer ride — but a **15% short-term tax flips the advantage negative** (-0.09), 25% takes it to -0.28, and the edge decays across eras. Real-but-thin & tax-eaten — Fragile, not Investable, not a Mirage (the drawdown protection does survive costs).